app/exports/routes.py

```
@@ -0,0 +1,71 @@
+from flask import send_file
+from weasyprint import HTML
+
+from app.modules.ical_exporter import ICalExporter
+from app.modules.task_manager import TaskManager
+from . import exports_bp
+
+ical_exporter = ICalExporter()
+task_manager = TaskManager()
+
+
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

Redefinition of unused `datetime` from line 1

Remove definition: `datetime`

(F811)